pkg/endpoints/endpoints.go

```diff
@@ -33,11 +33,15 @@ const (
 	PowerVS serviceID = "powervs"
 	// RC used to identify Resource-Controller service.
 	RC serviceID = "rc"
+	// TransitGateway  service.
+	TransitGateway serviceID = "TransitGateway"
+	// COS service.
+	COS serviceID = "COS"
```

Comment:
Is it required for this to be camelcase or we can have it in lowercase as other IDs to maintain uniformity.

Reply:
Changed to be consistent with existing endpoints.

Reply:
@Prajyot-Parab how is aws or other vendors are handling this?

Reply:
@mkumatag everything in smallcase

Reply:
for a reference, can you please record that here

Reply:
Ref - https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/5aa9d22533db1f24da9c4b9e667ee162a2cb52e1/pkg/cloud/services/secretsmanager/cloudinit.go#L25